wallet/src/main.rs

```diff
@@ -80,13 +151,25 @@ fn main() -> Result<(), Box<dyn error::Error>> {
     let default_drone_port = format!("{}", default.drone_port);
 
     let matches = app(crate_name!(), crate_description!(), crate_version!())
+        .arg({
+            let arg = Arg::with_name("config_file")
+                .short("c")
+                .long("config")
+                .value_name("PATH")
+                .takes_value(true)
+                .help("Configuration file to use");
+            if let Some(ref config_file) = *config::CONFIG_FILE {
+                arg.default_value(&config_file)
+            } else {
+                arg
+            }
+        })
         .arg(
             Arg::with_name("json_rpc_url")
                 .short("u")
                 .long("url")
-                .value_name("URL")
+                .value_name("URL:PORT")
```

Comment:
hmm, URL implies :PORT to me

Reply:
Yeah, I went back and forth on that. I thought this was slightly more explicit... as it will definitely not work if you just put http://testnet.solana.com
But i don't feel that strongly, and will revert if you prefer.

Reply:
> as it will definitely not work if you just put http://testnet.solana.com

yeah but that's the wrong URL :)
The correct URL includes a port, http://testnet.solana.com:8899.  Providing "URL:PORT" implies the string http://testnet.solana.com:8899:8899 to me

Reply:
🙇‍♀